test/teams_manager_test.rb

```diff
@@ -26,19 +26,8 @@ def test_count_of_teams_full_file
 
     assert_equal 32, team_manager.count_of_teams
   end
-  
```

Comment:
Agreed, the test for that method that was phased out is no longer needed either.  Nice cleanup.